erpnext/stock/reorder_item.py

```
@@ -305,8 +296,9 @@ def _log_exception(mr):
 
 				company_wise_mr.setdefault(company, []).append(mr)
 
-			except Exception:
-				_log_exception(mr)
+			except Exception as exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)